### PR TITLE
Removed overwrite flag for external storages, added 409,410 errors, u…

### PR DESCRIPTION
**Pull request by khound**
…pdated status API with error details

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: sample-code/ic-customized-workflow-app/IC_customized_workflow_using_cc_storage.sh [hunk 1]
```diff
@@ -44,8 +44,7 @@ curl -s $ic_endpoint \
     "href": "'$intermediate_output_url'",
     "mask": {
       "format":"soft"
-    },
-    "overwrite":true
```
Comment (noahhoffman) [inline sample-code/ic-customized-workflow-app/IC_customized_workflow_using_cc_storage.sh:48]: @khound we should leave the flag in for CC Storage examples

File: src/pages/code-sample/index.md [hunk 2]
```diff
@@ -193,8 +192,7 @@ curl -X POST \
     {
       "href":"<SIGNED_POST_URL>",
       "storage":"<storage>",
-      "type":"vnd.adobe.photoshop",
-      "overwrite":true
```
Comment (noahhoffman) [inline src/pages/code-sample/index.md:197]: @khound maybe change these lines, in all examples to  either
`"overwrite":<overwrite_if_applicable>` or
`"overwrite":<overwrite>`
to indicate the value is potentially available here.  Thoughts?

**Comment by noahhoffman** [comment]
👍

File: src/pages/code-sample/index.md (hunk 2 above)
Comment (khound) [reply src/pages/code-sample/index.md:197]: @noahhoffman For code sample page actually we changed all the examples with storage ==`external` as our docs are targeted for external customers. So I think better not to have `overwrite` flag here.

File: sample-code/ic-customized-workflow-app/IC_customized_workflow_using_cc_storage.sh (hunk 1 above)
Comment (khound) [reply sample-code/ic-customized-workflow-app/IC_customized_workflow_using_cc_storage.sh:48]: Yes. I can put "overwrite":<overwrite_if_applicable> here

**Pull request merged**